src/server/command/worldedit/deselect.zig

```diff
@@ -0,0 +1,21 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+
+pub const description = "Clears pos1 and pos2 of selection.";
+pub const usage = "/deselect";
+
+pub fn execute(args: []const u8, source: *User) void {
+	if(args.len != 0) {
+		source.sendMessage("#ff0000Too many arguments for command /deselect. Expected no arguments.", .{});
+		return;
+	}
+	source.mutex.lock();
+	defer source.mutex.unlock();
+
+	source.commandData.selectionPosition2 = null;
```

Comment:
Shouldn't this also deselect pos1? At least the command description says so.

Reply:
Indeed